StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetSnapshotTests.swift

```diff
@@ -37,6 +37,7 @@ class PaymentSheetSnapshotTests: STPSnapshotTestCase {
     var runAgainstLiveService: Bool = false
     override func setUp() {
         super.setUp()
+//        recordMode = true
```

Comment:
Note that you can rerun these by choosing the "AllStripeFrameworks-RecordMode" scheme and running the test, you don't need to add this line anymore

Reply:
Oooh thanks